.github/workflows/ci.yml

```diff
@@ -43,6 +43,66 @@ jobs:
         run: |
           ./Godot_v3.4-stable_linux_server.64 --path test -s script.gd
 
+  linux-cmake:
+    name: Build (Linux, GCC, CMake)
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get update -qq
+          sudo apt-get install -qqq build-essential pkg-config cmake
+          curl -LO https://downloads.tuxfamily.org/godotengine/3.4/Godot_v3.4-stable_linux_server.64.zip
```

Comment:
Shouldn't we use 3.4.2 instead?

Reply:
I just use same version scons CI uses. I could try to update although.

Reply:
@Calinou Looks like it works.